### PR TITLE
Bug 1195340 - Migrate siteIcon to the new gaia-site-icon component r=gmarty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=268961648526031a5dda60a89681b0eca6a1d440

File: apps/system/js/app_chrome.js [hunk 1]
```diff
@@ -1031,21 +1029,11 @@
    * @param {string?} url
    */
   AppChrome.prototype.setSiteIcon = function ac_setSiteIcon(url) {
-    var icon;
 
     if (!this.siteIcon || this.app.isPrivateBrowser()) {
       return;
     }
```
Comment (albertopq) [inline apps/system/js/app_chrome.js]: We already have the default Image as default in the css

File: apps/system/js/app_chrome.js [hunk 2]
```diff
@@ -7,12 +7,10 @@
 /* global SettingsListener */
 /* global Service */
 /* global GaiaPinCard */
-/* global Icon */
 
 'use strict';
 
 (function(exports) {
-  const DEFAULT_ICON_URL = '/style/chrome/images/default_icon.png';
```
Comment (albertopq) [inline apps/system/js/app_chrome.js]: We already have this Icon as default backgroundImage

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=27d4b7dcddb5bee5c6b5bb86aab4f8715bcc323f

**Pull request closed without merging**